tests/glhe/topology/test_glhe.py

```diff
@@ -1,6 +1,6 @@
 import unittest
 
-from glhe.topology.glhe import GLHE
```

Comment:
So `from glhe.borehole.glhe` was giving a circular import because the first glhe in that phrase was finding itself.  I changed the deep file name to full_ground_loop and everything was happy.

Reply:
Good catch!